docs/program/apis/sessions.md

```diff
@@ -23,6 +23,12 @@ Without session management, the API request from the client sets the flow of ele
 With default configuration, sessions are automatically managed for you with Viam's `SessionsClient`.
 If you want to manage sessions yourself, use Viam's sessions management API.
 
+The session management API moderates `viam-server`'s client network connections.
+You can think about a user as one of many clients that are connected to `viam-server` like a star in a solar system of application clients.
+All stars (clients) are subject to forces of the planet (`viam-server`) and the sun ( the Session Management API), with the API being the larger force managing viam-server's clients' connection to the robot.
```

Comment:
```suggestion
All stars (clients) are subject to forces of the planet (`viam-server`) and the sun (the Session Management API), with the API being the larger force managing viam-server's clients' connection to the robot.
```
Not reviewing just saw this since I'm tagged and noticed this extra space